### DISCUSSION TITLE
drop_openai_simple_evals_gen_3857b0.py中处理方式无法处理 drop数据集，脚本中的drop_simple_eval 数据集哪里能找到呢？

### DISCUSSION THREAD

**Discussion by KADCA21** [open-compass/opencompass#1394] [Q&A]
如题

**Comment by tonysy** [comment]
The latest version will download this dataset automaticly, also you can find the dataset url here.
https://github.com/open-compass/opencompass/blob/c81329b548e253c7057eeff7c4aac9e97ae32127/opencompass/utils/datasets_info.py#L325

**Reply by KADCA21** [reply]
Thank you for your response. And here's an another question, the results obtained from the drop_simple_eval dataset appear to be significantly higher than those on the DROP leaderboard, like Qwen-1.8B-chat, it gets 30+ on drop_simple_eval while the leaderboard shows it only get 5.8. Could you please let me know which dataset is used for evaluation on the leaderboard?

![image](https://github.com/user-attachments/assets/f2fdc96e-bacf-4fe0-a37e-448720a86a63)
